R/rules.R

```diff
@@ -120,3 +120,96 @@ as.list.rule <- function(x, ...) {
 
   r_list
 }
+
+#' Combine Two Rules
+#'
+#' @param x (`rule`) to modify.
+#' @param y (`rule`) rule whose mapping will take precedence over the ones described in `x`.
```

Comment:
the later one takes priority. seems this is quite normal and we also should use it in citril right

Reply:
Yes, here it is more natural since we are used to `modifyList`.